lib/themeroller.js

```diff
@@ -112,17 +112,33 @@ function ThemeRoller( baseThemeCss, vars, options ) {
 
 	} else {
 
-		// For version >= 1.10.0, filter has its own separate line and variable name.
-		opacityFix = function( opacity ) {
-			return ( opacity / 100 ).toString().replace( /^0\./, "." );
-		};
-		opacityFilter = function( opacity ) {
-			return "Alpha(Opacity=" + opacity + ")";
-		};
-		vars.opacityFilterOverlay = opacityFilter( vars.opacityOverlay );
-		vars.opacityFilterShadow = opacityFilter( vars.opacityShadow );
-		vars.opacityOverlay = opacityFix( vars.opacityOverlay );
-		vars.opacityShadow = opacityFix( vars.opacityShadow );
+		if ( options.version && semver.lt( options.version, "1.13.0" ) ) {
+
+			// For version >= 1.10.0 and < 1.13.0, filter has its own separate line and variable name.
+			opacityFix = function( opacity ) {
+				return ( opacity / 100 ).toString().replace( /^0\./, "." );
+			};
+			opacityFilter = function( opacity ) {
+				return "Alpha(Opacity=" + opacity + ")";
+			};
+			vars.opacityFilterOverlay = opacityFilter( vars.opacityOverlay );
+			vars.opacityFilterShadow = opacityFilter( vars.opacityShadow );
+			vars.opacityOverlay = opacityFix( vars.opacityOverlay );
+			vars.opacityShadow = opacityFix( vars.opacityShadow );
+		}
+		else {
+			// For version >= 1.13.0, -ms-filter has its own separate line and variable name.
+			opacityFix = function( opacity ) {
+				return ( opacity / 100 ).toString().replace( /^0\./, "." ) + ';-ms-filter: "alpha(opacity=' + opacity + ')"';
```

Comment:
This doesn't look right. `opacityFix` is applied to lines meant for modern browsers so it mostly needs to be divided by 100 but we shouldn't be adding `-ms-filter`. Such an addition was done for versions `<1.10.0` because they had it on the same line so were replaced together. In newer versions, the filter-based declarations for IE have separate variables to replace.

I think the `>=1.13.0` block should be closer to the current `>=1.10.0` one but with the updated `opacityFilter` definition.